ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1451,13 +1451,13 @@ CO2M:
   guid: "c73f80ff-e57e-4fe4-8bd6-26e4581b7269"
   description: "Basic carbon dioxide monitoring."
   is_abstract: true
+  opt_uses:
+  - high_zone_air_co2_concentration_alarm
+  - high_limit_zone_air_co2_concentration_setpoint
```

Comment:
The addition of a setpoint would make this more of a control type than a monitoring type. Would you be able to use `CO2CDSP` for your application and add the alarm as optional there?

Reply:
yes this could work